lib/main.js

```diff
@@ -882,8 +928,20 @@ function SetRenderer(renderer) {
   return MathJax.Hub.setRenderer(renderer);
 }
 
+function RerenderPHTML(result) {
+  if ((data.html || data.htmlNode || data.css) && (data.phtml || data.phtmlNode || data.pcss)) {
```

Comment:
I'm wondering if you could just check the `renderer` variable here:

```
if (renderer !== 'PreviewHTML' && (data.phtml || data.phtmlNode || data.pcss)) {
  renderer = 'PreviewHTML';
...
}
```

and make a similar change to the `RerenderSVG()` function below.